class ExitException(Exception):
    def __init__(self):
        pass


class ExitFunctionException(Exception):
    def __init__(self):
        pass
